platform/windows/detect.py

```diff
@@ -180,13 +180,18 @@ def get_opts():
             "MSVC version to use. Ignored if VCINSTALLDIR is set in shell env.",
             None,
         ),
+        ("DXC_PATH", "Path to the DirectX Shader Compiler distribution (required for D3D12)", ""),
+        ("AGILITY_SDK_PATH", "Path to the Agility SDK distribution (optional for D3D12)", ""),
+        ("agility_sdk_multiarch", "Whether the Agility SDK DLLs will be stored in arch-specific subdirectories", False),
+        ("PIX_PATH", "Path to the PIX runtime distribution (optional for D3D12)", ""),
```

Comment:
I would move these further down, after a `# Direct3D 12 support` comment to group them better.

I would also suggest using lower case names for those, which is more consistent with how we name SCons variables usually.

I don't want this to block merging, so could be done in a follow-up, as you prefer.

Reply:
Done. Coming in the next push.